lib/src/base-styles/_nx-colors.scss

```diff
@@ -157,6 +164,11 @@
     // NxCounter
     --nx-color-counter: var(--nx-swatch-indigo-90);
     --nx-color-counter-active-text: var(--nx-swatch-indigo-05);
+
+    // NxCollapsibleItems
+    --nx-color-collapsible-items-background-active: var(--nx-swatch-indigo-30);
```

Comment:
As per our discussion yesterday, we should be able to reuse this variable for other components like the Icon Only Btn & Pagination, so we could make this one `--nx-color-interactive-background-active'

Reply:
> As per our discussion yesterday, we should be able to reuse this variable for other components like the Icon Only Btn & Pagination, so we could make this one `--nx-color-interactive-background-active'

The issue with that is that `--nx-color-interactive-background-active` has already been defined for light mode (`grey-95`), but for collapsible items, the correct color is `grey-97` =/